components/application-mgt/org.wso2.carbon.identity.application.mgt.ui/src/main/resources/web/application/configure-service-provider.jsp

```diff
@@ -2285,4 +2324,46 @@
         </div>
     </div>
 
+    <div id = "showDialog" title = "WSO2 Carbon">
+        <h2 style="margin-left:20px;margin-top:20px;">
+            <fmt:message key="title.oauth.application"/>
+        </h2>
+
+        <% if ("add".equals(operation)) { %>
+            <p style="font-size: 12px;margin-top:6px;margin-left:20px;"><fmt:message key="application.successfull"/></p>
+        <% } %>
+
+        <% if ("regenerate".equals(action)) { %>
+            <p style="font-size: 12px;margin-top:6px;margin-left:20px;">
+                <% String message = MessageFormat.format(resourceBundle.getString("application.regenerated"), appBean.getOIDCClientId()); %>
+                <%= message %>
+            </p>
+        <% } %>
+
+        <div style="margin-left:20px;background-color: #f4f4f4; border-left: 6px solid #cccccc;height:50px;width:90%;">
+            <p style="margin:20px;padding-top:10px;display:block;"><strong>Note : <font color="red"><fmt:message key="note.oauth.application"/></font></strong></p>
+        </div>
+        <table style="margin-left:20px;margin-top:25px;">
+            <thead>
+                <tr style="height: 35px;">
+                    <th class="leftCol-small" style="font-size: 12px;"><fmt:message key="config.application.consumerkey"/></th>
+                    <td>
+                        <input style="border: none; background: white; font-size: 14px;" size="25" autocomplete="off" id="consumerKey" name="oauthConsumerKey" value=<%=appBean.getOIDCClientId()%> readonly="readonly">
+                        <span style="float: right;">
+                            <button onclick="return copyTextClick(document.getElementById('consumerKey'))" name="copyBtn" id="copyBtn"><fmt:message key="button.copy"/></button>
+                        </span>
+                    </td>
+                </tr>
+                <tr style="height: 35px;">
+                    <th class="leftCol-small" style="font-size: 12px;"><fmt:message key="config.application.consumersecret"/></th>
+                    <td>
+                        <input style="border: none; background: white;font-size: 14px;" size="25" autocomplete="off" id="consumerSecret" name="oauthConsumerSecret" value="<%=oauthConsumerSecret%>" readonly="readonly">
```

Comment:
Please use `<%=Encode.forHtmlAttribute(attributeName)%>` for all relevant fields